apps/email/style/compose_cards.css

```diff
@@ -88,6 +88,58 @@ input.cmp-subject-text {
   text-overflow: ellipsis;
 }
 
+/* Allows the position absolute autocomplete list to be relative to this
+   element */
+.cmp-autocomplete-origin {
+  position: relative;
+}
+
+/* Prevents scrolling of areas under the autocomplete list to avoid confusing
+   dueling scrollbars */
+.cmp-autocomplete-noscroll {
+  overflow: hidden;
+}
+
+/* When the autocomplete is active, change the flex alignment so that the label
+and + buttons are visible when typing */
+.cmp-autocomplete-noscroll .cmp-envelope-line {
+  align-items: flex-end;
+}
+
+.cmp-autocomplete-spacer {
+  background-color: #f2f2f2;
```

Comment:
going forward discussion: would it make sense to start using CSS variables for things like this where the color really wants to be updated in lock-step with its other uses?